emcmake.py

```diff
@@ -34,11 +34,10 @@ def has_substr(args, substr):
   # Append the Emscripten toolchain file if the user didn't specify one.
   if not has_substr(args, '-DCMAKE_TOOLCHAIN_FILE'):
     args.append('-DCMAKE_TOOLCHAIN_FILE=' + utils.path_from_root('cmake', 'Modules', 'Platform', 'Emscripten.cmake'))
-  node_js = config.NODE_JS
 
   if not has_substr(args, '-DCMAKE_CROSSCOMPILING_EMULATOR'):
     node_js = config.NODE_JS[0].replace('"', '\"')
```

Comment:
Hmmm..  I just noticed that this line does not do what the author thinks it does :-/   This is a no-op.   I will followup about that.